packages/commonwealth/client/scripts/views/pages/view_thread/ThreadPollCard.tsx

```diff
@@ -58,11 +62,12 @@ export const ThreadPollCard = ({
         })}
         incrementalVoteCast={1}
         isPreview={false}
-        tooltipErrorMessage={
-          app.user.activeAccount
-            ? null
-            : 'You must join this community to vote.'
-        }
+        tooltipErrorMessage={(() => {
```

Comment:
This does feel a little awkward, maybe extract it into it's own function rather than generating and invoking inline like this? Ofc, not a blocker. 

ex: 
const getTooltipErrorMessage = () => {
  if (!app.user.activeAccount) {
    return 'Error: You must join this community to vote.';
  }

  if (isTopicMembershipRestricted) {
    return 'Error: Topic is gated.';
  }

  return '';
};

`tooltipErrorMessage={getTooltipErrorMessage()}`

Reply:
done